rdkit/Chem/UnitTestEnumerateHeterocycles.py

```diff
@@ -149,20 +139,16 @@ def test_apply_every_rule_to_every_fragment(self):
             for src, rxn in zip(GetHeterocycleReactionSmarts(), GetHeterocycleReactions()):
                 for smiles in get_unique_products(rxn, rdkit_mol):
                     assert_valid_change(orig_can_smi, smiles)
-                    row = {
-                        'SMILES'  : orig_can_smi,
-                        'MUTATED' : smiles,
-                        'REACTION' : src.SMARTS + '>>' + src.CONVERT_TO,
-                        'DESCRIPTION' : src.DESCRIPTION,
-                        }
-                    writer.writerow(row)
+                    desc = {'SMILES'  : orig_can_smi, 'MUTATED' : smiles,
+                           'REACTION' : src.SMARTS + '>>' + src.CONVERT_TO,
+                           'DESCRIPTION' : src.DESCRIPTION,
+                           }
```

Comment:
Restore the original version. Dictionaries should either be all on one line or all key: value pairs on individual lines.